src/app-router.js

```diff
@@ -22,14 +23,7 @@ export class AppRouter extends Router {
 
   loadUrl(url) {
     return this.createNavigationInstruction(url)
-      .then(instruction => this.queueInstruction(instruction))
-      .catch(error => {
```

Comment:
This error handling is now rolled into `dequeueInstruction`.

I think I need to add something back here if `createNavigationInstruction` rejects, such as when a route is not found. We should restore the previous fragment in that case to make sure the document location remains consistent.